content/hardware/01.mkr/01.boards/mkr-wifi-1010/tutorials/mkr-battery-app-note/mkr-battery-app-note.md

```diff
@@ -0,0 +1,340 @@
+
+***Warning: Please make sure that the battery terminals do NOT touch each other. The short circuit can cause permanent damage to the battery and even explosion.***
+
+Apart from being a single cell (3.7V), a key factor to selecting the correct battery is that the connector is compatible.
+
+### Connector
+You can connect a battery to the MKR WIFI 1010 via a 2-pin JST-PH female connector. The PH variant of JST connectors are identified by a pin-to-pin distance of 2 mm. Here are several examples of LiPo batteries with a 2-pin JST-PH connector. Each individual connector is made of one plastic housing and two metal crimp terminals. A crimping device may be required. Note that when looking from above (with the notch facing you), the red (positive) wire should be on your left.
+
+![JST-PH connector connected to the MKR WiFi 1010. Note that the positive terminal (red wire) is to the left, towards the PMIC](assets/battery-in-mkr.png)
+You can connect a battery to the MKR WIFI 1010 via a 2-pin JST-PH female connector. The PH variant of JST connectors are identified by a pin-to-pin distance of 2mm. Here are several examples of LiPo batteries with a 2-pin JST-PH connector. Each individual connector is made of one plastic housing and two metal crimp terminals. A crimping device may be required. Note that when looking from above (with the notch facing you), the red (positive) wire should be on your left. 
+
+### Protection Circuit
+A protection circuit cuts off the battery if overcurrent or under/over voltage is detected. This adds an additional layer of safety.
+![LiPo Battery Breakdown](assets/lipo-battery-breakdown.png)
+
+### BQ24195L Power Management IC
+The BQ24195L IC is responsible for charging the single cell lion battery, as well as boosting the battery voltage to 3.3V for use on the board internally. An Arduino library ([Arduino_BQ24165](https://github.com/arduino-libraries/Arduino_BQ24195)) has been developed, to help control the features of this battery over I2C. 
+
+### Electrical Properties
+
+**Voltage**
+The nominal voltage of both LiPo and Li-Ion batteries is around 3.7V and is commonly referred to as the voltage of single cell batteries. However, this is an ideal scenario since as a battery is used the voltage changes. The voltage can be considered similar to a waterfall. The higher the waterfall, the higher the voltage.
+
+
+In the MKR boards, the battery terminal is connected to the SAMD21 via a reserved pin (PB09) known as `ADC_BATTERY` within the Arduino Core. This pin is used internally on the board, and is not accessible via the MKR pins. Since the voltage of a Li-ion battery exceeds 3.3V (the AREF value), a voltage divider must be used to extend the range while also ensure that only safe voltages are applied to the microcontroller. We can calculate the output voltage using the following formula
+
+$$ V_{out} = \frac{V_{source} \times R2} {R_1 + R_2} $$

```

Comment:
Hey @aliphys. Nicely written document for the MKR board. But I think I discovered a little bug here, which is the use of "$" as a styling component? It doesn't render anything at docs if you take a look at it, so I am not sure how you can render it differently? Like using `backticks` maybe?

Reply:
Thanks for catching the render issue. The `$` sign is used to render math in Tex. It renders OK locally.

@jhansson-ard What syntax do we use for Tex in the `docs-content` repo?